libs/ui-watt/src/lib/components/dropdown/watt-dropdown.component.html

```diff
@@ -36,7 +36,7 @@
     ></ngx-mat-select-search
   ></mat-option>
 
-  <mat-option *ngIf="!multiple">—</mat-option>
+  <mat-option *ngIf="!multiple && emptyOption">—</mat-option>
```

Comment:
suggestion: add test that verifies that the reset option can be removed. I can help with that if you want.

Reply:
Just pushed a couple of tests.

Reply:
So did I :D